.github/pull_request_template.md

```diff
@@ -13,7 +13,10 @@ Significant internal changes:
 
 Requirement changes:
 - Bulleted list, if relevant, of any changes to setup.py, requirement.txt, environment.yml, etc
-- See this [instructions](https://github.com/VulcanClimateModeling/fv3net#updating-dependencies) about how to update dependencies
+- [ ] Ran `make lock_deps/lock_pip`
+- [ ] Add PR review with license info for any additions to `constraints.txt`
+  ([example](https://github.com/VulcanClimateModeling/fv3net/pull/1218#pullrequestreview-663644359))
+- [ ] Follows these [instructions](https://vulcanclimatemodeling.com/docs/fv3net/dependency_management.html#dependency-management)
```

Comment:
The page has a fair bit of text to read as part of doing a review, but it looks like only the "adding or updating dependencies" section is needed reading? Can alter this as needed, or not change it at all if you don't think this is needed.

```suggestion
- [ ] Follows these [adding and updating dependencies instructions](https://vulcanclimatemodeling.com/docs/fv3net/dependency_management.html#dependency-management)
```

Reply:
The docs should probably recommend use of the `make lock_pip` rule if just updating the constraints.txt. Currently the docs only refer to the `make lock_deps` rule, which in the past I've been told to avoid since it causes large changes to the conda lock files.

Reply:
I made an issue for your suggestion @oliverwm1 .

Reply:
@nbren12 can we delete this last TODO now that the instructions are linked in the first one? it's redundant currently.

Reply:
oh whoops. I just deleted it.